zeppelin-web/src/app/notebook/notebook.controller.js

```diff
@@ -752,14 +752,43 @@ function NotebookCtrl ($scope, $route, $routeParams, $location, $rootScope,
 
   $scope.savePermissions = function () {
     convertPermissionsToArray()
+    if ($scope.isOwnerEmpty()) {
+      BootstrapDialog.show({
+        closable: false,
+        title: 'Setting Owners Permissions',
+        message: 'Please fill the [Owners] field. If not, it will set as current user.\n\n' +
+          'Current user : [ ' + $rootScope.ticket.principal + ']',
```

Comment:
@felixcheung thanks for carefully checking this.
I didn't care this `$rootScope.ticket.principal` variable, because this `$scope.savepermissions` function is not run when user doesn't use Siro and user didn't login. 
But, for prevent, I would be better to add `if statement`for checking about this case that `$rootScope.ticket.principal` is ayonymous or empty.